src/js/api/config.js

```diff
@@ -84,6 +84,13 @@ const Config = function(options, persisted) {
     config.language = language;
     config.intl = intl;
 
+    // If autoPause is configured with an empty block,
+    // default autoPause.viewability to true.
+    let autoPause = config.autoPause;
+    if (autoPause && !autoPause.viewability) {
```

Comment:
This doesn't work if I set-up with `autoPause: { viewability: false }`, as this block changes it to `true`.

Reply:
👍